### PR TITLE
changing  error message

### PR DESCRIPTION
**Pull request by jolanipekun**
John, I have changed the error message slightly. Thank you.

### CODE REVIEW

File: checkmypass.py [hunk 1]
```diff
@@ -33,7 +33,7 @@ def main(args):
     for password in args:
         count = pwned_api_check(password)
         if count:
-            print(f'{password} was found {count} times... you should probably change your password')
+            print(f'{password} was found {count} times... you should probably change your password, please.')
```
Comment (jolanipekun) [inline checkmypass.py:36]: Thank you, John for your review and text changes.

**Review by jolanipekun — COMMENTED** [review]
nice.

**Comment by libertyballmarkers** [comment]
HI BRO I HAVE A DEAL FOR YOU REGARDING CODING. YOU CAN DROP ME YOUR ICQ, SKYPE OR EMAIL.

**Comment by jolanipekun** [comment]
***@***.***

**Pull request merged**